Tasks/SeleniumWebDriverInstaller/seleniumwebdriverinstaller.ts

```diff
@@ -0,0 +1,133 @@
+        const firefoxDriver = tl.getInput('firefoxDriver', false);
+        const firefoxDriverVersion = tl.getInput('firefoxDriverVersion', false);
+        const ieDriver = tl.getInput('ieDriver', false);
+        const ieDriverVersion = tl.getInput('ieDriverVersion', false);
+        const edgeDriver = tl.getInput('edgeDriver', false);
+        const edgeDriverVersion = tl.getInput('edgeDriverVersion', false);
+
+        // Acquire and cache the user specified drivers.
+        if (chromeDriver === 'true') {
+            await acquireAndCacheDriverVersion(enums.Driver.ChromeWebDriver, chromeDriverVersion);
+        }
+        if (firefoxDriver === 'true') {
+            await acquireAndCacheDriverVersion(enums.Driver.GeckoWebDriver, firefoxDriverVersion);
+        }
+        if (ieDriver === 'true') {
+            await acquireAndCacheDriverVersion(enums.Driver.IEWebDriver, ieDriverVersion);
+        }
+        if (edgeDriver === 'true') {
+            await acquireAndCacheDriverVersion(enums.Driver.EdgeWebDriver, edgeDriverVersion);
+        }
+
+    } catch (error) {
+        tl.setResult(tl.TaskResult.Failed, error.message);
+    }
+}
+
+async function acquireAndCacheDriverVersion(driver: enums.Driver, driverVersion: string) {
+    let driverName = getDriverName(driver);
+    console.log(tl.loc('AcquiringDriver', driverName, driverVersion));
+
+    driverVersion = toolLib.cleanVersion(driverVersion);
+    if (driverVersion === undefined || driverVersion === null) {
+        throw new Error(tl.loc('InvalidVersion'));
+    }
+    let driverArch = getDriverArch(driver);
+    tl.debug(`Semantic version: ${driverVersion}, Architecture: ${driverArch}`);
+
+    let toolPath = toolLib.findLocalTool(driverName, driverVersion, driverArch);
+
+    if (!toolPath || toolPath === 'undefined') {
+        tl.debug(`Could not find version ${driverVersion} of driver ${driverName}, will download it.`)
+        let driverPath = getDownloadUrl(driver, driverVersion);
+        let downloadDir = path.join(tl.getVariable('Agent.TempDirectory'));
+        let downloadPath = path.join(downloadDir, getDriverExeName(driver));
+        try {
+            tl.debug('Attempting download from ' + driverPath + ' to ' + downloadPath);
+            await downloadutility.download(driverPath, downloadPath);
+            tl.debug('File downloaded to ' + downloadPath);
+        }
+        catch (error) {
+            tl.error(error.message);
+            throw new Error(tl.loc('FailedToAcquireDriver'));
+        }
+        toolPath = await toolLib.cacheDir(downloadDir, driverName, driverVersion, driverArch);
+        tl.debug("Driver has been cached to " + toolPath);
+    }
+    else {
+        tl.debug(`Found version ${driverVersion} of driver ${driverName} in cache.`)
+        console.log(tl.loc('FileExists', toolPath));
+    }
+
+    console.log(`Setting Environment variable ${driverName}=${toolPath}`);
```

Comment:
>console.log(`Setting Environment variable ${driverName}=${toolPath}`); [](start = 4, length = 70)

needs to be localized since you are logging and not debug